### PR TITLE
Initialize dropout differently across batch and across sequence in BE…

### PR DESCRIPTION
**Pull request by j5b**
…RT example.

### CODE REVIEW

**Review by patrick-kidger — COMMENTED** [review]
This bit:
```python
        ff_keys = (
            [None] * seq_len
            if ff_key is None
            else jax.random.split(ff_key, num=seq_len)
        )
```
looks wrong to me. I think it should be
```python
        ff_keys = (
            None
            if ff_key is None
            else jax.random.split(ff_key, num=inputs.shape[0])
        )
```

Other than that LGTM!

**Comment by j5b** [comment]
Good catch! This should be fixed now.

**Comment by patrick-kidger** [comment]
Great! Thanks for the fix.
